Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,446 @@
+        m_nextButton = m_backNextButtons->addButton(tr("Next"), QDialogButtonBox::ApplyRole);
+
+        connect(m_tabWidget, &TabWidget::currentChanged, this, &CreateAGemScreen::ConvertNextButtonToCreate);
+        connect(m_backButton, &QPushButton::clicked, this, &CreateAGemScreen::HandleBackButton);
+        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::HandleNextButton);
+
+        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::OnGemDisplayNameUpdated);
+        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::OnGemSystemNameUpdated);
+        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::OnLicenseNameUpdated);
+        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::OnCreatorNameUpdated);
+        connect(m_nextButton, &QPushButton::clicked, this, &CreateAGemScreen::OnRepositoryURLUpdated);
+
+        setObjectName("createAGemBody");
+        setLayout(vLayout);
+
+    }
+
+    QScrollArea* CreateAGemScreen::CreateFirstScreen()
+    {
+        QScrollArea* firstScreenScrollArea = new QScrollArea();
+        firstScreenScrollArea->setWidgetResizable(true);
+        firstScreenScrollArea->setMinimumWidth(660);
+        firstScreenScrollArea->setObjectName("createAGemRightPane");
+        QFrame* firstScreenFrame = new QFrame();
+
+        QWidget* firstScreenScrollWidget = new QWidget();
+        firstScreenScrollArea->setWidget(firstScreenScrollWidget);
+
+        QVBoxLayout* firstScreen = new QVBoxLayout();
+        firstScreen->setAlignment(Qt::AlignTop);
+        firstScreen->addWidget(firstScreenFrame);
+        firstScreenScrollWidget->setLayout(firstScreen);
+        QLabel* rightPaneHeader = new QLabel();
+        rightPaneHeader->setObjectName("rightPaneHeader");
+        rightPaneHeader->setText(tr("Please Choose a Gem Template"));
+        QLabel* rightPaneSubheader = new QLabel();
+        rightPaneSubheader->setObjectName("rightPaneSubheader");
+        rightPaneSubheader->setText(tr("Gems can contain assets such as scripts, animations, meshes, textures, and more."));
+        firstScreen->addWidget(rightPaneHeader);
+        firstScreen->addWidget(rightPaneSubheader);
+
+        QRadioButton* button1 = new QRadioButton("Default Gem");
+        button1->setObjectName("qRadioButton");
+        QRadioButton* button2 = new QRadioButton("Prebuilt Gem");
+        button2->setObjectName("qRadioButton");
+        QRadioButton* button3 = new QRadioButton("Asset Gem");
+        button3->setObjectName("qRadioButton");
+        QRadioButton* button4 = new QRadioButton("C++ Tool Gem");
+        button4->setObjectName("qRadioButton");
+        QRadioButton* button5 = new QRadioButton("Python Tool Gem");
+        button5->setObjectName("qRadioButton");
+        QRadioButton* button6 = new QRadioButton("Choose existing template");
+        button6->setObjectName("qRadioButton");
```

Comment:
```suggestion
        QRadioButton* button2 = new QRadioButton("Prebuilt Gem");
        QRadioButton* button3 = new QRadioButton("Asset Gem");
        QRadioButton* button4 = new QRadioButton("C++ Tool Gem");
        QRadioButton* button5 = new QRadioButton("Python Tool Gem");
        QRadioButton* button6 = new QRadioButton("Choose existing template");
```
You can remove the custom object name and use the existing one.  Also, these shouldn't be hardcoded - OK for now - the list of gem templates should eventually be calculated by exposing existing functionality that lists templates in PythonBindingsInterface - similar to how templates are generated in `NewProjectSettingsScreen::AddTemplateButtons`, but we need a new method in `PythonBindings` that can pull the gem templates.

Reply:
Fixed.